corehq/middleware.py

```diff
@@ -336,7 +330,7 @@ def get_view_func(view_fn, view_kwargs):
         try:
             class_name = dispatcher.get_report_class_name(domain, slug)
             return to_function(class_name) if class_name else None
-        except:
+        except Exception:
```

Comment:
My ignorance to report dispatchers is pretty bad. Do we still have custom report dispatchers/is this try/except still relevant?

Reply:
Looks like it, `CustomProjectReportDispatcher` is still around.